manas_cafa5/protein.py

```diff
@@ -44,6 +46,17 @@ def go_terms(self):
             self.load_uniprot()
         return self.terms.get('go')
 
+    def go_terms_children(self, graph):
+        return list(reduce(
+            lambda terms, term: terms.union(networkx.descendants(graph, term['id'])),
+            self.go_terms(),
+            set()
+        ))
+
+    def build_graph(url_or_file):
```

Comment:
This should be a [`@staticmethod`](https://python-reference.readthedocs.io/en/latest/docs/functions/staticmethod.html).